### ISSUE TITLE
[lulzfuzz] cmd_info segfaults

### ISSUE THREAD

**Issue by zonkzonk** [radare/radare2#6437]
morrn,

```
Fri Jan  6 00:04:40 CET 2017
Reading symbols from /usr/local/bin/radare2...done.

warning: core file may not match specified executable file.
[New LWP 17742]
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Core was generated by `r2 -qn -'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x00007ff8ec7cc21f in cmd_info (data=0x55ac2623a640 <r>, input=0x7ff8ec882240 "z") at cmd_info.c:416
416                                     RBININFO ("strings", R_CORE_BIN_ACC_STRINGS, NULL, r_list_length (obj->strings));
(gdb) l
411                                             mode = (input[2] == 'q')
412                                                     ? R_CORE_BIN_SIMPLEST
413                                                     : R_CORE_BIN_SIMPLE;
414                                             input++;
415                                     }
416                                     RBININFO ("strings", R_CORE_BIN_ACC_STRINGS, NULL, r_list_length (obj->strings));
417                             }
418                             break;
419                     case 'c': // for r2 `ic`
420                             if (input[1]== '?') {
(gdb) bt
#0  0x00007ff8ec7cc21f in cmd_info (data=0x55ac2623a640 <r>, input=0x7ff8ec882240 "z") at cmd_info.c:416
#1  0x00007ff8ec7ccb7d in cmd_info (data=0x55ac2623a640 <r>, input=0x55ac274ea7c2 "a  %1269168516 %1073741824\\n") at cmd_info.c:503
#2  0x00007ff8ec83290d in r_cmd_call (cmd=0x55ac274b8ca0, input=0x55ac274ea7c0 "ipa  %1269168516 %1073741824\\n") at cmd_api.c:226
#3  0x00007ff8ec7f818b in r_core_cmd_subst_i (core=0x55ac2623a640 <r>, cmd=0x55ac274ea7c0 "ipa  %1269168516 %1073741824\\n", colon=0x0) at cmd.c:2063
#4  0x00007ff8ec7f57cf in r_core_cmd_subst (core=0x55ac2623a640 <r>, cmd=0x55ac274ea7c0 "ipa  %1269168516 %1073741824\\n") at cmd.c:1341
#5  0x00007ff8ec7fa044 in r_core_cmd (core=0x55ac2623a640 <r>, cstr=0x55ac27468070 "ipa  %1269168516 %1073741824\\n", log=1) at cmd.c:2619
#6  0x00007ff8ec78519a in r_core_prompt_exec (r=0x55ac2623a640 <r>) at core.c:1825
#7  0x000055ac2603748c in main (argc=3, argv=0x7fff401d1868, envp=0x7fff401d1888) at radare2.c:1094
(gdb) q
Fri Jan  6 00:05:20 CET 2017
Reading symbols from /usr/local/bin/radare2...done.

warning: core file may not match specified executable file.
[New LWP 6825]
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Core was generated by `r2 -qn -'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x00007f2a72a4175c in cmd_info (data=0x564e3f108640 <r>, input=0x564e4077f7c1 "cl  %1269168516 %1073741824\\n") at cmd_info.c:472
472                                             playMsg (core, "classes", r_list_length (obj->classes));
(gdb) l
467                                                             break;
468                                                     }
469                                                     goto done;
470                                             }
471                                     } else {
472                                             playMsg (core, "classes", r_list_length (obj->classes));
473                                             if (input[1] == 'l' && obj) { // "icl"
474                                                     r_list_foreach (obj->classes, iter, cls) {
475                                                             r_list_foreach (cls->methods, iter2, sym) {
476                                                                     const char *comma = iter2->p? " ": "";
(gdb) 
``` 

Greetings
--zlul

**Comment by radare** [comment]
cant reproduce. in fact.. i dont know which commands are you typing here..

**Comment by alvarofe** [comment]
Check master again I think is a null dereference on obj otherwise I am lost

**Comment by zonkzonk** [comment]
@radare ipa %1269168516 %1073741824\\n

**Comment by radare** [comment]
wat

**Comment by leberus** [comment]
$ r2 --
 -- EIP = 0x41414141
[0x00000000]> ipa %1269168516 %1073741824\n
No file selected
Segmentation fault (core dumped)
$

I'll try to get my head around

**Comment by radare** [comment]
thats a silly null deref, i will push the fix as soon as my compiler finished buliding r2 in this machiine.

reproducer is just r2 -qcia —

no need for all that compilcated and meaningless command

**Comment by radare** [comment]
Fixed in e5c38591c6005f615cbc192800593241aae19a62
